packages/samples/headless-ssr/app-router/package.json

```diff
@@ -10,7 +10,7 @@
     "prod": "next start",
     "lint": "next lint",
     "e2e": "cypress run --browser chrome --e2e --project ../",
-    "e2e:watch": "cypress open --browser chrome --e2e --project ../"
+    "e2e:watch": "cypress open --browser chrome --e2e --project ../ --env NEXTJS_ROUTER=app"
```

Comment:
guess this is not used in the tests - but can leave it for consistency and any future requirement